builtin/settingtypes.txt

```diff
@@ -721,7 +721,7 @@ contentdb_url (ContentDB URL) string https://content.minetest.net
 #    as defined by the Free Software Foundation.
 #    You can also specify content ratings.
 #    These flags are independent from Minetest versions,
-#    so see a full list at https://content.minetest.net/help/content_flags/
+#    so, see a full list at https://content.minetest.net/help/content_flags/
```

Comment:
Huh?

Reply:
I am guessing that the spell checker only sees it as a new sentence (new line), hence there is a comma after the word _so_. I can not find more information/rule about placing a comma after _so_ etc.

Reply:
Yeah there shouldn't be a comma there, the spell checker is probably confused over the linebreak